ansible/roles/grafana-dashboards/tasks/main.yml

```diff
@@ -108,17 +108,51 @@
       when:
         - grafana_dashboards | length > 0
 
-    - name: import grafana dashboards
-      community.grafana.grafana_dashboard:
-        grafana_url: "{{ grafana_api_url }}"
-        grafana_user: "{{ grafana_security.admin_user }}"
-        grafana_password: "{{ grafana_security.admin_password }}"
-        path: "{{ _tmp_dashboards.path }}/{{ ((item.dashboard_id | string) + '.json') if 'dashboard_id' in item else item.dashboard_file }}"
-        # message renamed to commit_message (Be aware if using old ansible)
-        # https://github.com/ansible/ansible/pull/60051
-        #commit_message: Updated by ansible
-        state: present
-        overwrite: true
-      #no_log: true
-      with_items: "{{ grafana_dashboards }}"
-      when: grafana_state | default('started') != 'stopped'
+- name: Create/Update dashboards file (provisioning)
```

Comment:
@m-bull this is /basically/ https://github.com/cloudalchemy/ansible-grafana/blob/master/tasks/dashboards.yml#L108, modified to run on the grafana node rather than on the deploy host.